### PR TITLE
Add full report for xml

### PR DESCRIPTION
**Pull request by lyriccoder**
fixing #320

### CODE REVIEW

File: aibolit/__main__.py [hunk 1]
```diff
@@ -210,44 +166,49 @@ def calculate_patterns_and_metrics(file):
...
         sorted_result = predict(input_params, model, args)
-        code_lines = None
         patterns_list = model.features_conf['patterns_only']
-        pattern_code = None  # type: ignore
         for iter, (key, val) in enumerate(sorted_result.items()):
             if key in patterns_list:
                 pattern_code = key
                 code_lines = code_lines_dict.get('lines_' + key)
-                # We show only positive gradient, we won't add patterns
+                # We show only positive gradient
```
Comment (acheshkov) [inline aibolit/__main__.py:196]: we do no have gradient anymore

**Comment by 0crat** [comment]
This pull request [#321](https://github.com/yegor256/aibolit/issues/321) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/62586217-970b-4940-bbc1-bb663dba62f8); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Comment by acheshkov** [comment]
@rultor release, tag=`1.0.1`

**Pull request merged**